src/controllers/travelControllers.js

```diff
@@ -1,115 +1,108 @@
+import socketEmission from '../services/websocket';
 import {
-  onewayTripService,
-  showManagerPendingAppr,
-  showUsertravelsStatus,
-  approveTravel,
-  mostTraveled
+    onewayTripService,
+    showManagerPendingAppr,
+    showUsertravelsStatus,
+    approveTravel,
 } from '../services/travelServices';
 import { findUserByEmail } from '../services/authServices';
 import { successResponseWithData, errorResponse } from '../utils/response';
+import sendVerificationEmail from '../utils/email';
 
 import message from '../utils/messageUtils';
 import statusCode from '../utils/statusCode';
 
-export const createOneWayTrip = async (req, res) => {
-  try {
-    const user = await findUserByEmail(req.userData.email);
-    const { id, dept_id } = user.dataValues;
-
-    const data = await onewayTripService({
-      user_id: id,
-      travel_type: 'one-way',
-      ...req.body,
-      dept_id,
-    });
-
-    successResponseWithData(
-      res,
-      statusCode.created,
-      message.oneWayTripCreated,
-      data
-    );
-  } catch (err) {
-    errorResponse(res, statusCode.serverError, err);
-  }
+const { emission } = socketEmission;
+
+export const createOneWayTrip = async(req, res) => {
+    try {
+        const user = await findUserByEmail(req.userData.email);
+        const { id, email, dept_id } = user.dataValues;
+
+        if (!dept_id) {
+            errorResponse(res, statusCode.badRequest, message.lineManager);
+        }
+
+        const data = await onewayTripService({
+            user_id: id,
+            travel_type: 'one-way',
+            ...req.body,
+            dept_id,
+        });
+
+        const emailVerify = await sendVerificationEmail(email,
+            'Travel Confirmation', message.notifyUser);
+        emission('here', 'we made it');
```

Comment:
Please, what message are you emitting here?

Reply:
> Please, what message are you emitting here?

That was me testing the feature however it haas been updated